source/win-main/EditorPane.vue

```diff
@@ -11,18 +11,20 @@
       v-bind:leaf-id="leafId"
       v-bind:window-id="windowId"
     ></DocumentTabs>
-    <MainEditor
-      v-if="activeFile !== null"
-      ref="editor"
-      v-bind:distraction-free="distractionFree"
-      v-bind:leaf-id="leafId"
-      v-bind:window-id="windowId"
-      v-bind:editor-commands="editorCommands"
-      v-on:global-search="$emit('globalSearch', $event)"
-    ></MainEditor>
-    <div v-else class="empty-pane"></div>
+    <template v-for="file in openFiles" v-bind:key="file.path">
+      <MainEditor
+        v-show="activeFile?.path === file.path"
+        v-bind:file="file"
+        v-bind:distraction-free="distractionFree"
+        v-bind:leaf-id="leafId"
+        v-bind:window-id="windowId"
+        v-bind:editor-commands="commandsForFile(file)"
+        v-on:global-search="$emit('globalSearch', $event)"
+      ></MainEditor>
+    </template>
+    <!-- Show a placeholder if there are no open files -->
+    <div v-if="openFiles.length === 0" class="empty-pane"></div>
```

Comment:
We could move the length check to a _computed_ prop with a descriptive name, which would then make the comment above obsolete.

Reply:
Done.